src/main/java/sample/Application.java

```diff
@@ -0,0 +1,10 @@
+package sample;
```

Comment:
Since these classes are released under the Apache License, will you please add that as a header to each Java class? I've just added a sample header to the [contributing guidelines](https://github.com/spring-projects-experimental/spring-authorization-server/blob/master/CONTRIBUTING.adoc#apache-license-header) as a reference.